include/exadg/convection_diffusion/postprocessor/postprocessor.cpp

```diff
@@ -49,6 +49,18 @@ PostProcessor<dim, Number>::setup(Operator<dim, Number> const & pde_operator)
                          pp_data.output_data);
 }
 
+template<int dim, typename Number>
+void
+PostProcessor<dim, Number>::setup_after_coarsening_and_refinement(
+  Operator<dim, Number> const & pde_operator)
```

Comment:
Here, I would like to skip the argument `pde_operator`. This is also an object that stays valid during AMR. Currently, the postprocessor in the `ConvDiff` module does not store a pointer to this operator, but other modules like `IncNS` already do this. So I think we should do something similar here.

Reply:
The argument is not needed since the whole funciton is not needed?

Reply:
I just checked `IncNS::PostProcessor` and there we do in `setup(Operator const & pde_operator)`:
`navier_stokes_operator = &pde_operator;`
So I understand your comment in the sense that we should skip the `pde_operator` argument for the ConvDiff module.